v2/googlecloud-to-elasticsearch/docs/BigQueryToElasticsearch/README.md

```diff
@@ -61,7 +82,7 @@ echo '{
     "image":"'${TARGET_GCR_IMAGE}'",
     "metadata":{
       "name":"BigQuery to Elasticsearch",
-      "description":"Replicates BigQuery data into an Elasticsearch index",
+      "description":"Ingests BigQuery data into an Elasticsearch index",
       "parameters":[
```

Comment:
For below parameters isOptional should be changed to true
inputTableSpec, elasticsearchUsername, elasticsearchPassword

ApiKey should be added to Json spec as mandatory parameter